doc/esmvalcore/preprocessor.rst

```diff
@@ -138,344 +260,559 @@ losing generality. To mask ice out, `mask_landseaice` can be used:
 
-1. extract_volume
------------------
 
-This function extracts a specific range in the z-direction from a cube.
-This function takes two arguments:
+``extract_volume``
+------------------
 
-* z_min: minimum in the z direction
-* z_max: maximum in the z direction
+Extract a specific range in the `z`-direction from a cube.  This function
+takes two arguments, a minimum and a maximum (``z_min`` and ``z_max``,
+respectively) in the `z`-direction.
 
-Note that this requires the requested z-coordinate range to be the same sign as
-the Iris cube, i.e. if the cube has z-coordinate as negative, then z_min and
-z_max need to be negative numbers.
+Note that this requires the requested `z`-coordinate range to be the same sign
+as the Iris cube. ie, if the cube has `z`-coordinate as negative, then
+``z_min`` and ``z_max`` need to be negative numbers.
 
 See also :func:`esmvalcore.preprocessor.extract_volume`.
 
-2. volume_statistics
---------------------
+
+``volume_statistics``
+---------------------
 
 This function calculates the volume-weighted average across three dimensions,
 but maintains the time dimension.
 
-This function takes one argument:
-* operator: operation to apply over the volume (at the moment only mean is implemented)
+This function takes the argument: ``operator``, which defines the operation to
+apply over the volume. 
 
-No depth coordinate is required as this is determined by Iris. This
-function works best when the fx files provide the cell volume.
+No depth coordinate is required as this is determined by Iris. This function
+works best when the ``fx_files`` provide the cell volume.
 
 See also :func:`esmvalcore.preprocessor.volume_statistics`.
 
 
-3. depth_integration
---------------------
+``depth_integration``
+---------------------
 
-This function integrates over the depth dimension. It performs a weighted sum
-along the z-coordinate, and removes the z direction of the output cube. It takes no arguments.
+This function integrate over the depth dimension. This function does a weighted
+sum along the `z`-coordinate, and removes the `z` direction of the output
+cube. This preprocessor takes no arguments.
 
 See also :func:`esmvalcore.preprocessor.depth_integration`.
 
-4. extract_transect
--------------------
 
-This function extracts data along a line of constant latitude or longitude.
-This function takes two arguments, although only one is strictly required:
-* latitude
-* longitude
+``extract_transect``
+--------------------
 
-One of these arguments needs to be set to a float, and the other can then be
-either ignored or set to a minimum or maximum value. For example, if latitude
-is set to 0 and longitude is left blank, the function would produce a cube
-along the equator. If latitude is set to to 0 and longitude to `[40., 100.]` it
-will produce a transect of the equator in the Indian Ocean.
+This function extract data along a line of constant latitude or longitude.
```

Comment:
Should be "This function exctracts..." instead of "This function extract..."?

Reply:
Done.